Backend/app/rider-platform/rider-app/Main/spec/API/ticket.yaml

```
@@ -335,4 +377,46 @@ apis:
+      response:
+        type: API.Types.UI.TicketService.TicketAssignmentResp
+
+  # GET /ticket/bookings/{ticketServiceShortId}/ticket-assignment
+  - GET:
+      endpoint: /ticket/bookings/{ticketServiceShortId}/ticket-assignment
+      name: getTicketBookingsTicketAssignment
+      auth: TokenAuth
+      params:
+        ticketServiceShortId: ShortId TicketBookingService
+      response:
+        type: API.Types.UI.TicketService.TicketAssignmentResponse
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)